app.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'book'
+require_relative 'label'
+require_relative 'book_methods'
+require_relative 'music'
+require_relative 'music_album_display'
+require_relative 'game'
+require_relative 'game_author'
+require 'json'
+
+class App
+  def main_menu
+    puts "Welcome to our catalog 👋\nSelect an option: 👉"
+    options = [
+      '--------------------------------------------------',
+      '1 - Books catalog',
+      '    *************************',
+      '2 - Music albums catalog',
+      '    *************************',
+      '3 - Games catalog',
+      '    *************************',
+      '4 - Quit',
+      '--------------------------------------------------'
+    ]
+    puts options
+    option = gets.chomp.downcase
+    case option
+    when '1'
+      book = BookMethods.new('./data/books.json')
+      book.run
```

Comment:
- [ ] Please investigate the `Book` option once more to see where this error is coming from. 🙏 
```
Traceback (most recent call last):
        5: from app.rb:3:in `<main>'
        4: from app.rb:3:in `require_relative'
        3: from C:/Users/mbakr/OneDrive/Desktop/Ruby-kanban/book_methods.rb:2:in `<top (required)>'
        2: from C:/Users/mbakr/OneDrive/Desktop/Ruby-kanban/book_methods.rb:2:in `require_relative'
        1: from C:/Users/mbakr/OneDrive/Desktop/Ruby-kanban/app.rb:48:in `<top (required)>'
C:/Users/mbakr/OneDrive/Desktop/Ruby-kanban/app.rb:28:in `main_menu': uninitialized constant App::Bo
okMethods (NameError)
```

Reply:
> - [ ] Please investigate the `Book` option once more to see where this error is coming from. 🙏 
> ```
> Traceback (most recent call last):
>         5: from app.rb:3:in `<main>'
>         4: from app.rb:3:in `require_relative'
>         3: from C:/Users/mbakr/OneDrive/Desktop/Ruby-kanban/book_methods.rb:2:in `<top (required)>'
>         2: from C:/Users/mbakr/OneDrive/Desktop/Ruby-kanban/book_methods.rb:2:in `require_relative'
>         1: from C:/Users/mbakr/OneDrive/Desktop/Ruby-kanban/app.rb:48:in `<top (required)>'
> C:/Users/mbakr/OneDrive/Desktop/Ruby-kanban/app.rb:28:in `main_menu': uninitialized constant App::Bo
> okMethods (NameError)
> ```

 Can you please demonstrate the steps you have done to get this error?

Reply:
@MahmoudBakr23 can you please demonstrate what have you done to produce this error?

Reply:
First run ruby app.rb
Second, choose number 1
Then I got the error

Reply:
@MahmoudBakr23 That's because you have to use the main entry point "main.rb", run main.rb, this works completely fine.

Reply:
I think you meant the menu.rb file not main.rb, because you don't have a main.rb file in your code base

Reply:
I renamed it to main.rb, but yeah I meant menu.rb